index.md

```diff
@@ -69,15 +78,7 @@ Blog post
 Read about the recent PyMC-Data Umbrella sprint in this interview with
 Sandra Meneses, one of the participants who submitted a PR
 :::
-:::{grid-item-card} PyMC 4.0 is closing in
-:link: v4_announcement
-:link-type: ref
-:class-header: bg-pymc-one
 
-Release announcement
-^^^
-Check out this blog post for a sneak preview of all the improvements PyMC 4.0 will bring!
-Or install the already available beta release and try it for yourself!
 :::
```

Comment:
this is the closing of the grid item card element and also needs to be removed, otherwise in trying to close it sphinx generates an extra empy card: https://pymcio--14.org.readthedocs.build/#announcements